plugins/plugin-openwhisk/src/lib/cmds/openwhisk-core.ts

```diff
@@ -1166,29 +1149,10 @@ const executor = (commandTree, _entity, _verb, verbSynonym?) => async ({ argv: a
         .then(options => ow[entity][verb](options))
         .then(handle204(options.name))
         .then(postprocess)
-        .then(response => {
-          // amend the history entry with a selected subset of the response
-          if (execOptions && execOptions.history) {
-            historyModel.update(execOptions.history, entry => {
-              entry.response = { activationId: response.activationId }
-              return entry.response
-            })
-          }
-          return response
-        })
         .then(pretty)
         .then(correctMissingBindingName(options))
         .then(response => Array.isArray(response) ? Promise.all(response.map(pretty)) : response)
         .then(response => Array.isArray(response) && response.length > 0 ? withHeader(response, execOptions) : response)
-        .then(response => {
-          if (commandTree &&
-              (!execOptions || !execOptions.noHistory || (execOptions && execOptions.contextChangeOK)) && // don't update context for nested execs
-              (response.verb === 'get' || response.verb === 'create' || response.verb === 'update')) {
-            return response
-          } else {
-            return response
-          }
-        })
```

Comment:
The lines above seem outdated because the branch here doesn't do anything rather than simply return the response.